src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommand.cs

```diff
@@ -76,6 +77,23 @@ public async Task<CreateDialogResult> Handle(CreateDialogCommand request, Cancel
             dialog.Org = serviceResourceInformation.OwnOrgShortName;
         }
 
+        if (request.IdempotentKey is not null && !string.IsNullOrEmpty(dialog.Org))
+        {
+            var dialogId = await _db.Dialogs
+                .Where(x => x.IdempotentKey == request.IdempotentKey && x.Org == dialog.Org)
```

Comment:
Nit: The order of the checks should probably match that of the composite index definition. However, as we're using a simple comparison, the order of the checks here should be handled by the optimizer in PostgreSQL such that it doesn't make any difference. For a inequality check (>, <, BETWEEN), the order _would_ be significant, so I think it's generally best to always match the order.